programs/token-metadata/program/src/lib.rs

```diff
@@ -10,6 +10,7 @@ pub mod escrow {
     pub use crate::{instruction::escrow::*, processor::escrow::*};
 }
 
+#[cfg(not(feature = "no-entrypoint"))]
```

Comment:
Discussed offline: BPF condition check redundant, because you would always be doing no-entrypoint as a dependency lib, and you would only be doing not no-entrypoint when its an an on-chain program.